django/backends/db/auth.go

```diff
@@ -75,7 +75,7 @@ func (p *permission) Code() string {
 
 type user struct {
 	DID        int64  `db:"id" json:"id"`
-	DUsername  string `db:"username" json:"-"`
+	DUsername  string `db:"phone" json:"-"`
```

Comment:
Nopes. We have to write generic code, this is specific to Acko.

Reply:
Plan:

1. Remove all non id fields from `user` struct. 
2. Add `extra = map[string]interface{}` to user struct.
3. Deserialize into `extra`.
4. Initializae DID and extra
5. In types.go: User, add `.Field(string) (interface{}, bool)`,  `.SetField(string, interface{}) bool` (if field doesn't exist in extra, it can be set). 
6. Helpers for all types: .Int()/.SetInt() and so on for String, Float, Bool, Date.
7. Update User.Save() to include all `extra`.
8. `User.RefreshFromDB()`.